# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fenced-frame/embedder-require-corp.https.html.ini
# Commit: 8339bdf8fcc8
# Full Hash: 8339bdf8fcc85604672251b5c26800f2638e69d2
# Author: Cosmin Sabou <csabou@mozilla.com>
# Date: 2023-04-29 09:20:24
# Description:
#   Bug 1829504 - Mark fenced-frame tests as intermittent on win and linux debug. r=aryx
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D176769
# ==============================================================================

diff -r 48f93c057e50 -r 8339bdf8fcc8 testing/web-platform/meta/fenced-frame/embedder-require-corp.https.html.ini
--- a/testing/web-platform/meta/fenced-frame/embedder-require-corp.https.html.ini	Fri Apr 28 17:51:25 2023 +0000
+++ b/testing/web-platform/meta/fenced-frame/embedder-require-corp.https.html.ini	Fri Apr 28 17:26:15 2023 +0000
@@ -1,8 +1,7 @@
 [embedder-require-corp.https.html]
   expected:
-    if (os == "linux") and debug and fission and not swgl: [ERROR, CRASH]
-    if (os == "win") and debug and (processor == "x86_64") and not swgl: [CRASH, ERROR]
-    if (os == "linux") and debug and not fission: [ERROR, CRASH]
+    if (os == "linux") and debug: [ERROR, CRASH]
+    if (os == "win") and debug: [CRASH, ERROR]
     ERROR
   [Create fencedframe with COEP:require-corp]
     expected: TIMEOUT